packages/libsail/libsail.0.18/opam

```diff
@@ -0,0 +1,69 @@
+opam-version: "2.0"
+synopsis:
+  "Sail is a language for describing the instruction semantics of processors"
+description: """
+Sail is a language for describing the instruction-set
+architecture (ISA) semantics of processors. Sail aims to provide a
+engineer-friendly, vendor-pseudocode-like language for describing
+instruction semantics. It is essentially a first-order imperative
+language, but with lightweight dependent typing for numeric types and
+bitvector lengths, which are automatically checked using Z3. It has
+been used for several papers, available from
+http://www.cl.cam.ac.uk/~pes20/sail/.
+"""
+maintainer: ["Sail Devs <cl-sail-dev@lists.cam.ac.uk>"]
+authors: [
+  "Alasdair Armstrong"
+  "Thomas Bauereiss"
+  "Brian Campbell"
+  "Shaked Flur"
+  "Jonathan French"
+  "Kathy Gray"
+  "Robert Norton"
+  "Christopher Pulte"
+  "Peter Sewell"
+  "Mark Wassell"
+]
+license: "BSD-2-Clause"
+homepage: "https://github.com/rems-project/sail"
+bug-reports: "https://github.com/rems-project/sail/issues"
+depends: [
```

Comment:
Somewhere a lower bound is missing:
```

#=== ERROR while compiling libsail.0.18 =======================================#
# context              2.3.0~alpha~dev | linux/x86_64 | ocaml-base-compiler.4.08.1 | file:///home/opam/opam-repository
# path                 ~/.opam/4.08/.opam-switch/build/libsail.0.18
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p libsail -j 39 --promote-install-files=false @install
# exit-code            1
# env-file             ~/.opam/log/libsail-7-110e29.env
# output-file          ~/.opam/log/libsail-7-110e29.out
### output ###
# File "/home/opam/.opam/4.08/lib/biniou/biniou.dune", line 1, characters 0-0:
# Warning: .dune files are ignored since 2.0. Reinstall the library with dune
# >= 2.0 to get rid of this warning and enable support for the subsystem this
# library provides.
# (cd _build/default && /home/opam/.opam/4.08/bin/ocamlc.opt -w -40 -w -33 -w -27 -g -bin-annot -I src/lib/.libsail.objs/byte -I /home/opam/.opam/4.08/lib/biniou -I /home/opam/.opam/4.08/lib/dune-private-libs/dune-section -I /home/opam/.opam/4.08/lib/dune-site -I /home/opam/.opam/4.08/lib/easy-format -I /home/opam/.opam/4.08/lib/lem -I /home/opam/.opam/4.08/lib/lem_zarith -I /home/opam/.opam/4.08/lib/linksem -I /home/opam/.opam/4.08/lib/linksem_zarith -I /home/opam/.opam/4.08/lib/menhirLib -I /home/opam/.opam/4.08/lib/pprint -I /home/opam/.opam/4.08/lib/yojson -I /home/opam/.opam/4.08/lib/zarith -no-alias-deps -open Libsail -o src/lib/.libsail.objs/byte/libsail__Project_parser.cmi -c -intf src/lib/project_parser.mli)
# File "src/lib/project_parser.mli", line 47, characters 68-75:
# 47 | val file: (Lexing.lexbuf -> token) -> Lexing.lexbuf -> (Project.def spanned list)
#                                                                          ^^^^^^^
# Error: Unbound type constructor spanned
# (cd _build/default/src/lib && /home/opam/.opam/4.08/bin/ott -sort false -generate_aux_rules true -o ast.lem -picky_multiple_parses true ../../language/sail.ott)
# Ott version 0.32   distribution of Wed 9 Mar 16:05:28 GMT 2022
# (cd _build/default/src/lib && /home/opam/.opam/4.08/bin/ott -sort false -generate_aux_rules true -o jib.lem -picky_multiple_parses true ../../language/jib.ott)
# Ott version 0.32   distribution of Wed 9 Mar 16:05:28 GMT 2022
# (cd _build/default && /home/opam/.opam/4.08/bin/ocamlc.opt -w -40 -w -33 -w -27 -g -bin-annot -I src/lib/.libsail.objs/byte -I /home/opam/.opam/4.08/lib/biniou -I /home/opam/.opam/4.08/lib/dune-private-libs/dune-section -I /home/opam/.opam/4.08/lib/dune-site -I /home/opam/.opam/4.08/lib/easy-format -I /home/opam/.opam/4.08/lib/lem -I /home/opam/.opam/4.08/lib/lem_zarith -I /home/opam/.opam/4.08/lib/linksem -I /home/opam/.opam/4.08/lib/linksem_zarith -I /home/opam/.opam/4.08/lib/menhirLib -I /home/opam/.opam/4.08/lib/pprint -I /home/opam/.opam/4.08/lib/yojson -I /home/opam/.opam/4.08/lib/zarith -intf-suffix .ml -no-alias-deps -open Libsail -o src/lib/.libsail.objs/byte/libsail__Parser.cmo -c -impl src/lib/parser.ml)
# File "src/lib/parser.ml", line 21126, characters 19-36:
# 21126 |     Printf.fprintf Pervasives.stderr "Internal failure -- please contact the parser generator's developers.\n%!";
#                            ^^^^^^^^^^^^^^^^^
# Alert deprecated: module Stdlib.Pervasives
# Use Stdlib instead.
# 
# If you need to stay compatible with OCaml < 4.07, you can use the 
# stdlib-shims library: https://github.com/ocaml/stdlib-shims
# File "src/lib/parser.ml", line 2676, characters 12-22:
# 2676 |         let visibility = () in
#                    ^^^^^^^^^^
# Warning 26: unused variable visibility.
# File "src/lib/parser.ml", line 15941, characters 12-22:
# 15941 |         let visibility = () in
#                     ^^^^^^^^^^
# Warning 26: unused variable visibility.
# File "src/lib/parser.ml", line 20792, characters 8-18:
# 20792 |     let visibility = () in
#                 ^^^^^^^^^^
# Warning 26: unused variable visibility.
# (cd _build/default && /home/opam/.opam/4.08/bin/ocamlopt.opt -w -40 -w -33 -w -27 -g -I src/lib/.libsail.objs/byte -I src/lib/.libsail.objs/native -I /home/opam/.opam/4.08/lib/biniou -I /home/opam/.opam/4.08/lib/dune-private-libs/dune-section -I /home/opam/.opam/4.08/lib/dune-site -I /home/opam/.opam/4.08/lib/easy-format -I /home/opam/.opam/4.08/lib/lem -I /home/opam/.opam/4.08/lib/lem_zarith -I /home/opam/.opam/4.08/lib/linksem -I /home/opam/.opam/4.08/lib/linksem_zarith -I /home/opam/.opam/4.08/lib/menhirLib -I /home/opam/.opam/4.08/lib/pprint -I /home/opam/.opam/4.08/lib/yojson -I /home/opam/.opam/4.08/lib/zarith -intf-suffix .ml -no-alias-deps -open Libsail -o src/lib/.libsail.objs/native/libsail__Parser.cmx -c -impl src/lib/parser.ml)
# File "src/lib/parser.ml", line 21126, characters 19-36:
# 21126 |     Printf.fprintf Pervasives.stderr "Internal failure -- please contact the parser generator's developers.\n%!";
#                            ^^^^^^^^^^^^^^^^^
# Alert deprecated: module Stdlib.Pervasives
# Use Stdlib instead.
# 
# If you need to stay compatible with OCaml < 4.07, you can use the 
# stdlib-shims library: https://github.com/ocaml/stdlib-shims
# File "src/lib/parser.ml", line 2676, characters 12-22:
# 2676 |         let visibility = () in
#                    ^^^^^^^^^^
# Warning 26: unused variable visibility.
# File "src/lib/parser.ml", line 15941, characters 12-22:
# 15941 |         let visibility = () in
#                     ^^^^^^^^^^
# Warning 26: unused variable visibility.
# File "src/lib/parser.ml", line 20792, characters 8-18:
# 20792 |     let visibility = () in
#                 ^^^^^^^^^^
# Warning 26: unused variable visibility.
```

I am not sure where.